src/app/containers/CpsAssetMediaPlayer/index.jsx

```diff
@@ -37,16 +37,20 @@ const Wrapper = styled(GridItemConstrainedLargeNoMargin)`
     margin-top: ${GEL_SPACING_QUAD};
   }
 
-  figure {
+  ${props =>
+    !props.bottomPadding &&
+    `figure {
     padding-bottom: 0;
-  }
```

Comment:
Are we having to do this because a Psammead component is making assumptions about where it's being used?

Reply:
By default the psammed Figure component adds a 24px/1.5rem bottom padding and the media player is wrapped in the figure, this is the default experience.

MAP's have the media player at the top of page and do not want the extra bottom padding provided by figure, so they added this override, which by default is used across all pages that use the `cpsAssetMediaPlayer`.

The addition of the `bottomPadding` prop ensures that the padding override is now controlled by a prop and the MAP's now pass in this props to explicitly disable the padding.